store/store.go

```diff
@@ -178,6 +178,9 @@ type Config struct {
 
 	DetailFields []string
 	DeltaFormat  string
+
+	// CacheItemCount is the amount of downloads that should be cached
+	CacheItemCount int
```

Comment:
Can we rename this to CacheDownloads?

Also, perhaps s/amount/number/ in the comments.